build/vgg10-radioml/build.py

```diff
@@ -32,17 +32,33 @@
 import os
 import shutil
 
+import onnx
+from qonnx.core.modelwrapper import ModelWrapper
+
 # custom steps
 from custom_steps import step_pre_streamline, step_convert_final_layers
 
 model_name = "radioml_w4a4_small_tidy"
+model_file = "models/%s.onnx" % model_name
+
+verif_en = os.getenv("VERIFICATION_EN", "0")
 
 # which platforms to build the networks for
 zynq_platforms = ["ZCU104"]
 alveo_platforms = []
 platforms_to_build = zynq_platforms + alveo_platforms
 
 
+def custom_step_update_model(model, cfg):
```

Comment:
See previous comments

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)